tasks/release.py

```diff
@@ -1340,3 +1341,64 @@ def check_previous_agent6_rc(ctx):
         payload = {'message': err_msg}
         send_slack_msg(ctx, payload, os.environ.get("SLACK_DATADOG_AGENT_CI_WEBHOOK"))
         raise Exit(message=err_msg, code=1)
+
+
+@task
+def pin_integrations_core(ctx, slack_webhook=None):
+    """
+    Create a PR to bump the integrations core fields in the release.json file
+    """
+    if os.environ.get("GITHUB_ACTIONS"):
```

Comment:
I suppose you should include a default to avoid a `KeyError` when not running on the CI

Reply:
`get` return `None` by default. `["GITHUB_ACTIONS"]` would raise a `KeyError`

Reply:
Indeed, I got confused somewhere while reading the python doc, my bad!